suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Button, Card, Stack, Text } from '@suite-native/atoms';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { enableAnalyticsThunk } from '@suite-native/analytics';
+
+import { OnboardingScreen } from '../components/OnboardingScreen';
+import { AnalyticsInfoRow } from '../components/AnalyticsInfoRow';
+
+const buttonsWrapperStyle = prepareNativeStyle(() => ({
+    width: '95%',
+    alignSelf: 'center',
+}));
+
+const PrivacyDescription = () => (
+    <Text variant="hint" color="textSubdued">
+        We value privacy and security above all. Learn more about our data and security protocols{' '}
+        {/* Hack to vertically center the text with touchable opacity. */}
+        <Link href="https://trezor.io/" style={{ marginTop: -3 }}>
+            <Text color="textPrimaryDefault" variant="hint">
+                {' '}
```

Comment:
Extra whitespace. It is applied already in the outer Text.

Reply:
[411b2ef](https://github.com/trezor/trezor-suite/pull/7889/commits/411b2ef4d48df39092779f8f118b3b7d57b00cc0)